slicebase/bin/slice-update

```diff
@@ -1,9 +1,14 @@
 #!/bin/bash
 
 if [ -e /vsys/slice_update.out ] ; then
-    cat /vsys/slice_update.out &
+    cat /vsys/slice_update.out &> /dev/null & disown
     echo "1" > /vsys/slice_update.in
 else
-    echo "Error: /vsys/slice_update.out missing."
+    if [[ $EUID -ne 0 ]] ; then
```

Comment:
Shouldn't this test be the first conditional in the file?  If the user is not 0 and the conditional on line 3 is true, then presumably we'd get a permission denied error when it tries to write to /vsys/slice_update.in, right?